src/commandDetails/coin/leaderboard.ts

```diff
@@ -64,7 +70,7 @@ const getCoinLeaderboardEmbed = async (
         .join('\\_')
         .split('`')
         .join('\\`');
-      const userCoinEntryText = `${rank}. ${cleanUserTag} - ${
+      const userCoinEntryText = `${rank}\\. ${cleanUserTag} - ${
```

Comment:
Maybe here leave a comment to explain what the double backslash is doing (ie embeds using Markdown) as a reference, in case we run into anything similar in the future